smithy-build/src/test/java/software/amazon/smithy/build/model/SmithyBuildConfigTest.java

```diff
@@ -205,7 +205,7 @@ public void loadsFromNodeIgnoringBadSourceLocations() throws IOException {
                 .expectObjectNode();
         SmithyBuildConfig config = SmithyBuildConfig.fromNode(value);
 
-        Path cwd = Paths.get(".");
+        Path cwd = Paths.get(".").normalize().toAbsolutePath();
```

Comment:
Same as above, we can use here `SmithyBuildUtils.getCurrentWorkingDirectory()`.

Reply:
Was able to use it here. Updated.